content/Watsonx/CodeAssistant/101.md

```diff
@@ -0,0 +1,78 @@
+---
+title: '101: Ansible Lightspeed Environment Setup'
+timeToComplete: 15
+updated: 2023-09-13
+---
+
+# 101: Ansible Lightspeed environment setup in VS Code
+
+## Goal
+
+The goal of this lab is to install the Ansible Lightspeed extension and enable and authenticate the extension in VS Code. 
+
+## Install Ansible extension in VS Code
+
+1. Open the VS Code application.
+2. Click the **Extensions** icon on the left sidebar.
+
+    ![](./images/settings-icon.png)
+3. Search for **Ansible** in the search box and then click the **Install** button.
+
+    ![](./images/search-for-ansible.png)
+
+## Enable the Ansible Lightspeed extension
+
+1. After installing the Ansible extension, click on the little gear icon and select the **Extension Setting** option.
+
+    ![](./images/gear-icon.png)
+2. In the Search settings bar, type **lightspeed**
+3. If they are not already selected, check all of the following boxes:
+ - Ansible > Lightspeed: **Enabled**
+ - Ansible > Lightspeed > Suggestions: **Enabled**.
+
+    ![](./images/enable-lightspeed.png)
+
+**Note:** For the purpose of this demo, we will disable Ansible Lint and Ansible Execution Environment.
+
+1. In the Search settings bar, type **ansible lint**. If it is not already unselected, *uncheck* the **Ansible > Validation > Lint: **Enabled**** box.
+
+    ![](./images/disable-ansible-lint.png)
+
+2. In the Search settings bar, type **ansible execution**. If it is not already unselected, *uncheck* the **Ansible > Execution Environment: **Enabled**** box.
+
+    ![](./images/disable-ansible-exec.png)
+
+## Authenticate Ansible Lightspeed with GitHub
+
+1. Open the Ansible extension icon on the left-hand side of the Visual Studio Code interface and click **Connect**.
+
+    ![](./images/connect-ansible.png)
+
+2. Follow the prompts to sign in to Ansible Lightspeed with IBM watsonx Code Assistant using your GitHub credentials.
+
+    a. Click on the **Allow** button.
+
+    ![](./images/ansible-sign-in.png)
+
+    b. Click on the **Open** button.
+
+    ![](./images/external-website.png)
+
+    c. Click on the **Log in with GitHub** button.
+
+    ![](./images/github-login.png)
```

Comment:
Lets update this image to move the red box for this image to the "log in with GitHub" button.  Currently the instruction and the screenshot seem contradictory.